src/Xamarin.Android.Tools.AndroidSdk/OS.cs

```diff
@@ -139,9 +140,89 @@ internal static class RegistryEx
 		static extern int RegCreateKeyEx (UIntPtr hKey, string subKey, uint reserved, string? @class, uint options,
 			uint samDesired, IntPtr lpSecurityAttributes, out UIntPtr phkResult, out Disposition lpdwDisposition);
 
+		// https://docs.microsoft.com/en-us/windows/win32/api/winreg/nf-winreg-regenumkeyexw
+		[DllImport (ADVAPI, CharSet = CharSet.Unicode, SetLastError = true)]
+		static extern int RegEnumKeyExW (
```

Comment:
What is the need for p/invoke here? Is it an issue with 32-bit processes reading 64-bit keys?

Reply:
I'm not sure of the original need; this code *never* used `Microsoft.Win32.Registry` & co, and I'm not sure why.  (Apparently a question for @mhutch?)

Thus, given that we're already a tangled mess of P/Invokes, I figured it "best" to continue with the current paradigm.